src/vcpkg/platform-expression.cpp

```diff
@@ -139,26 +142,114 @@ namespace vcpkg::PlatformExpression
             // platform-expression =
             // | platform-expression-not
             // | platform-expression-and
-            // | platform-expression-or ;
+            // | platform-expression-or
+            // | platform-expression-low-precedence-or ;
```

Comment:
We need to write the real syntax; this isn't fully correct.

Reply:
(probably in vcpkg)

Reply:
Thanks. I will incorporate your suggested changes (in vcpkg) and update this code.

Reply:
thanks mark!

Reply:
after a few iterations, proposed EBNF update is [posted](https://github.com/microsoft/vcpkg/pull/21444), additional tests added and comments updated